backend/main.py

```
@@ -0,0 +1,190 @@
+)
+
+
+# Dummy Data fakedb
+fake_spots_db = []
+fake_user_db = {
+    "username": "walker_tk",
+    "coins": 1500,
+    "skin_id": 1
+}
+
+# 認証のフリをするための設定（実際は何もチェックしない）
```

Ruff (0.14.7)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)